bin/create.js

```diff
@@ -150,18 +167,36 @@ module.exports = Cli.createCommand('create', 'Create webtasks', {
                 if (argv.exp) parseDate(argv, 'exp');
                 
                 if (argv.secret) parseHash(argv, 'secret');
+                else             argv.secret = Object.create(null);
+
                 if (argv.param) parseHash(argv, 'param');
+                else            argv.param = Object.create(null);
+
                 if (argv.tokenLimit) parseHash(argv, 'tokenLimit');
                 if (argv.containerLimit) parseHash(argv, 'containerLimit');
-                
+
+                if (argv.auth0 && argv.share)
+                    throw new Error('Cannot specify both --share and --auth0');
+
+                if(argv.auth0 || argv.auth0 === '') {
+                    var auth0Credentials = argv.auth0.split(' ');
+
+                    argv.auth0 = {
+                        clientId: auth0Credentials[0]     || process.env.AUTH0_CLIENT_ID,
+                        clientSecret: auth0Credentials[1] || process.env.AUTH0_CLIENT_SECRET,
+                        domain: auth0Credentials[2]       || process.env.AUTH0_DOMAIN,
+                    };
```

Comment:
I'm not sure about this. Previously they were individual options, but since all three necessarily need to be specified together it adds verbosity in the user commands and the code. I think having `--auth0` is nice from a branding perspective as well, so I've left it for now.